admin_scripts/ubuntu/protect_settings.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -ex
+
+if [ !  -f "/usr/bin/gnome-control-center.real" ]
+then
+    cp  /usr/bin/gnome-control-center /usr/bin/gnome-control-center.real
```

Comment:
This is fragile: if the `gnome-control-center` package ever gets upgraded, then our patched version will be replaced.

Consider using `dpkg-divert --add` to make `dpkg` redirect all operations on `/usr/bin/gnome-control-center` to the version with the `.real` suffix.